x-pack/plugins/security_solution/public/flyout/right/hooks/use_accordion_state.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useReducer } from 'react';
+
+const CLOSED = 'closed' as const;
+const OPEN = 'open' as const;
+const toggleReducer = (state: unknown) => {
```

Comment:
since state can only be open or closed, consider creating a type to replace `unknown`